### ISSUE TITLE
Validation Accuracy remains constant

### ISSUE THREAD

**Issue by Dishank99** [VedantYadav/VQA#7]
I trained you model for 10epochs first and then for 25 epochs
One thing I noticed is that with every epoch trainig accuracy increases but the validation accuracy remains alomost constant
and also the prediction results are unexpected.

Can you please suggest something to resolve this issue

**Comment by seadark4** [comment]
@Dishank99 Hi, I believe you have succeeded in running this project. I managed to run it, but it took so long to generate the answer. And the website the author gave to download the VGG_16 weights is not available now, so I download another vgg16_weights online. I would appreciate it if you could upload your vgg16_weights.h5 on google drive.
